src/components/StudentsTable/StudentsTable.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { DataGrid } from '@material-ui/data-grid';
+
+
+const StudentsTable = ({rows, columns}) => (
+    <div style={{ height: 620, width: '100%' }}>
+      <DataGrid rows={rows} columns={columns} pageSize={10} checkboxSelection/>
```

Comment:
В компонентах лучше убирать захардкоженые  свойства и передавать пропсами,  можно добавить в пропсы значение по умолчанию



```suggestion
const StudentsTable = ({rows, columns,height=620,width='100'},pageSize=10) => (
...
<div style={{ height, width }}>
    <DataGrid rows={rows} columns={columns} pageSize={pageSize}/>
```